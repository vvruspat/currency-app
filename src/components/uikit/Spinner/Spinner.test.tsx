import { render, screen } from "@testing-library/react";
import { Spinner } from ".";

test("renders spinner", () => {
  render(<Spinner data-testid="test-spinner" />, {});
  const spinnerElement = screen.getByTestId("test-spinner");
  expect(spinnerElement).toBeInTheDocument();
  expect(spinnerElement).toMatchSnapshot();
});
