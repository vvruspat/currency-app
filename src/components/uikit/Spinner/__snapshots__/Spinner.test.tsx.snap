// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner 1`] = `
<div
  class="spinner-wrapper"
  data-testid="test-spinner"
>
  <div
    class="spinner"
  />
</div>
`;
